Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -7,31 +12,198 @@ public static class BalanceStats
         /// </summary>
         public static string FindHighestBalanceEver(string[] peopleAndBalances)
         {
-            return "";
+            if (peopleAndBalances == null || peopleAndBalances.Length == 0) return "N/A.";
+
+            int highestBalance = 0;
+            List<String> listOfPeople = new List<String>();
+
+            foreach (var values in peopleAndBalances)
+            {
+                // Splitting the values
+                var array = values.Split(", ");
```

Comment:
This may work in the unit test data set but this is going to be extremely fragile in any other situation as you are encoding whitespace requirements into the input data that may not map to any such restriction in data generation. In other words it could just as easily be `10,11,12` as `10, 11, 12`.

Reply:
Used a split & then a Trim afterwards